src/components/matching/MatchingCockpit.tsx

```diff
@@ -7,14 +9,14 @@ interface MatchingCockpitState {
 }
 
 const MATCHES_PLACEHOLDER = {
-  matches: [
+  matches: Array(5).fill(
```

Comment:
Please move this test data out to the component's test. We'll soon connect this component to the store and won't need this anymore and will forget about it.

Reply:
I'll need more hints to move this out to a test